pkg/monitortests/node/kubeletselinuxlabels/util.go

```diff
@@ -0,0 +1,74 @@
+package kubeletselinuxlabels
+
+import (
+	"github.com/MakeNowJust/heredoc"
+	"github.com/openshift/origin/test/extended/util/image"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	serviceAccountName = "selinuxlabel"
+)
+
+const (
+	busyScript = `
+#!/bin/bash
+
+set -eou pipefail
+
+SUB='kubelet_t'
+
+while true; do
+	kubelet_label=$(ps -Z -C kubelet)
```

Comment:
This doesn't work. :(

The shells that we were using don't have -C kubelet. Its why I went with the ugly version of awk.